src/routes/expenses.js

```diff
@@ -5,14 +5,14 @@ const {
 } = require('../controller/expensesController');
 
 const express = require('express');
+const { expensesService } = require('../services/expenses');
 
 // #region expensesRouter
 const expensesRouter = express.Router();
 
 function expences(app) {
   app.use('/expenses', expensesRouter);
-  expensesController.setInitialExpenses();
-  expensesRouter.use(express.json());
+  expensesService.setInitialExpenses();
```

Comment:
```suggestion
```

You don't need this method, cause every time you start server in calls `new expensesService()`. It call constructor, where you set empty array.

Reply:
it not works like you write and want. if we talk about server close/open it will work, but in tests. 
`beforeEach(() => {
    server = createServer();
    api = supertest(server);
  });`
  we just rerun function createServer(), so I need to create my objects with methods and with data inside this function call, or functions that I call in it.
  but Controllers method needs call service methods, so I need to make connection between them and in the same time they need to be in one function block to be recreated every time tests run different package of tests. 
  OR I make MVC separated files or I put them in one. 
  Once more test restart function - not server! so class methods will not be restarted, I need to recreate methods every time in routes! It's very stupid. We tell about this on classes, in QnA chanel. That's why I need here this function, so please also look logic of tests.
  
  createServer() => users() - routes + expenses() - routes - here I mention functions that restarts every test. and in expenses I need methods from user service....
  Every server start my code will create new instance of classes. But TESTS doesn't restart server, I cann't connect 4 separate class object and create them in one of routes function. I need to clean up data every test, I did it and I have no idea how to avoid clean up method